packages/framework/request-handler/src/requestHandlers.ts

```diff
@@ -34,7 +34,7 @@ export type RuntimeRequestHandler = (request: RequestParser, runtime: IContainer
  * handling of external URI to internal handle is required (in future, we will support weak handle references,
  * that will allow any GC policy to be implemented by container authors.)
  */
-export const deprecated_innerRequestHandler = async (request: IRequest, runtime: IContainerRuntime) =>
+export const deprecated_innerRequestHandler = async (request: IRequest, runtime: IContainerRuntimeBase) =>
```

Comment:
```suggestion
export const deprecated_innerRequestHandler = async (request: IRequest, runtime: IContainerRuntime & IContainerRuntimeBase) =>
```
this is a less breaking version of the change, given it is in a release

Reply:
actually it looks like IContainerRuntime implements IContainerRuntimeBase, so i'm not sure why we need this change at all.